nutanix/resource_nutanix_virtual_machine.go

```diff
@@ -1363,9 +1363,14 @@ func resourceNutanixVirtualMachineUpdate(d *schema.ResourceData, meta interface{
 			return fmt.Errorf("internal error: cannot shut down the VM with UUID(%s): %s", d.Id(), err)
 		}
 		// SpecVersion has changed due previous poweroff, increasing it manually (without reading the value again)
-		mySpec := *metadata.SpecVersion
-		mySpec += 2
-		metadata.SpecVersion = &mySpec
+		// mySpec := *metadata.SpecVersion
```

Comment:
can you revove these line if ther are not needed?

Reply:
Great catch! Can you verify again?